### PR TITLE
Fix ordered list markdown on CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by trenpixster**

### CODE REVIEW

**Comment by hbrls** [comment]
What are those 2 spaces green square?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ possible.
 ... (in particular how this is synced with internal changes to the project)
 
 ## Pull Requests
-We actively welcome your pull requests.
```
Comment (zpao) [inline CONTRIBUTING.md]: We just need a newline here. We don't need to do anything fancy with trailing whitespace.

Comment (trenpixster) [inline CONTRIBUTING.md]: As far as I can tell, adding new lines adds a bigger interval between the ordered list items. IMO the rendered markdown looks better with the trailing double whitespace. Would you rather have the extra newline instead?

**Comment by zpao** [comment]
I just did what I said in 35451459795e24acef22e290673f2f9dec1db5fc along with something else. Sorry for the delay (it was just way easier to fix this myself)

**Comment by trenpixster** [comment]
Oh! Sorry, I didn't get what you meant by the newline :+1:

**Comment by zpao** [comment]
No worries, thanks for catching this in the first place!

**Pull request closed without merging**